packages/mosaique/mosaique.0.0.1/opam

```diff
@@ -0,0 +1,41 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "OCaml bindings for libvips image processing library"
+description:
+  "OCaml bindings for libvips providing image loading, saving, and transformation operations"
+maintainer: ["Naora"]
+authors: ["Naora"]
+license: "MIT"
+homepage: "github.com/Naora/mosaique"
+bug-reports: "github.com/Naora/mosaique/issues"
+depends: [
+  "ocaml"
+  "dune-configurator"
+  "dune" {>= "3.20"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+x-maintenance-intent: ["(latest)"]
+depexts: [ 
+  ["libvips-dev"]{ os-family = "debian" | os-family = "ubuntu" } 
+]
```

Comment:
Thanks!

To follow the opam-repository's layout, I suggest splitting this part off in a separate package
`conf-libvips` (or `conf-vips`) stating the dependencies for each relevant platform and distribution,
and then adding a dependency under `depends` for that new `conf-` package.
That way, 
- we can add and improve support across platforms in `conf-libvips`/`conf-vips` independently and in parallel to
- you developing, improving, and releasing the bindings in `mosaique`

For example, here is the `conf-libflac` opam package, which we can use as a template:
https://github.com/ocaml/opam-repository/blob/master/packages/conf-libflac/conf-libflac.1/opam
(don't worry about not supporting all the platforms - we are happy help!) :slightly_smiling_face: